torchgeo/models/unet.py

```diff
@@ -39,6 +39,46 @@ class Unet_Weights(WeightsEnum):  # type: ignore[misc]
     .. versionadded:: 0.8
     """
 
+    UMBRA_GEC_OPENEARTHMAP_SAR = Weights(
+        url='https://hf.co/torchgeo/umbra_gec_unet_effb4_openearthmap_sar/resolve/738e0216fa5d41d14f535cc28430052683704142/umbra_gec_unet_effb4_openearthmap_sar-f21df2ee.pth',
+        transforms=K.AugmentationSequential(
+            K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)), data_keys=None
+        ),
+        meta={
+            'dataset': 'OpenEarthMap-SAR',
+            'in_chans': 1,
+            'num_classes': 9,
+            'model': 'U-Net',
+            'encoder': 'efficientnet-b4',
+            'classes': (
```

Comment:
This is a lot more metadata than other models. Is this coming from STAC MLM?

Reply:
Most of our weights are foundation models with no class specific outputs. I think it makes sense to add this here for the user to have information about the output order and categories.